web-ui/src/main/resources/catalog/views/default/directives/partials/mdactionmenu.html

```diff
@@ -332,7 +332,7 @@
       </li>
       <li role="menuitem" data-ng-repeat="f in formatterList">
         <a
-          data-ng-href="../api/records/{{md.uuid}}{{f.url.replace('${lang}', lang)}}{{f.url.indexOf('?') !== -1 ? '&': '?'}}approved={{mdView.current.record.draft != 'y'}}"
+          data-ng-href="{{buildFormatter(f.url,md.uuid,mdView.current.record.draft)}}"
```

Comment:
One time binding should be good enough (to avoid extra calls)
```suggestion
data-ng-href="{{::buildFormatter(f.url, md.uuid, mdView.current.record.draft)}}"
```